x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/mock_activity_log.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EndpointAction } from '../../../../../common/endpoint/types';
+
+export const mockActivityLogResponse = (agentId: string): EndpointAction[] => {
+  const activityLog: EndpointAction[] = [
```

Comment:
ℹ️  we also now have an Action generator, case that helps here in minimizing amount of mocks. See:

https://github.com/elastic/kibana/blob/57f59bd15dbb2075c4d8c7b93ab9bb4920248d6a/x-pack/plugins/security_solution/common/endpoint/data_generators/fleet_action_generator.ts#L15

If you do end up using it, I would suggest you initialize the class instance with a `seed` (argument to the class constructor)

Reply:
done 6a2cace04620e43ac4d3a4d57d52045fe5a5f7aa